tox.ini

```diff
@@ -1,130 +1,52 @@
 [tox]
-envlist = py37-{black,mypy,flake8,bandit,pylint,unit,integration}
+envlist = py3{7,8,9,10}
```

Comment:
Why is this still here if you also have it in `pyproject.toml`? I would choose one location and stick with it, else you're having to keep two sets of tox configuration data in sync.

Reply:
tox.ini deleted and shifting everything to the toml.